scripts/start.sh

```diff
@@ -3,11 +3,22 @@
 set -e
 
 for arg in "$@"; do
+  if [ "$arg" == "--debug" ]; then
+    IS_DEBUG=true
+    shift
+  fi
+
   if [ "$arg" == "--ship-logs" ]; then
     export LOCAL_LOG_SHIPPING=true
     shift
   fi
 done
 
+EXTRA_SBT_ARGS=""
+if [[ $IS_DEBUG == true ]]; then
+  EXTRA_SBT_ARGS="-jvm-debug 5005"
+fi
+
 yarn build-dev &
-sbt run
+# shellcheck disable=SC2086
```

Comment:
[shellcheck](https://github.com/koalaman/shellcheck/wiki/SC2086) would suggest this be `sbt "$EXTRA_SBT_ARGS" run`, which won't be correct in this case, so disable that rule.